tools/net/qemu-ifdown.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+# Delete the TAP interface.
+#
+# It's used for the cleanup script of QEMU netdev, DO NOT run it manually.
+
+if [ -n "$1" ]; then
+    ip link set dev "$1" down
```

Comment:
Do you think it's better to add another `ip link del` line to match the file description "Delete the TAP interface" (also it's a bit more consistent with `qemu-ifup.sh`)?